packages/perseus-editor/src/components/graph-locked-figures/locked-function-settings.tsx

```diff
@@ -0,0 +1,246 @@
+                            onChange={handlePropChange.bind(
+                                null,
+                                "directionalAxis",
+                            )}
+                            aria-label="equation prefix"
+                            style={styles.equationPrefix}
+                            // Placeholder is required, but never gets used.
+                            placeholder=""
+                        >
+                            <OptionItem value="x" label="y =" />
+                            <OptionItem value="y" label="x =" />
+                        </SingleSelect>
+                        <Strut size={spacing.xSmall_8} />
+                        <TextField
+                            type="text"
+                            aria-label="equation"
+                            value={equation}
+                            onChange={handlePropChange.bind(null, "equation")}
+                            style={[styles.textField]}
+                        />
+                    </View>
+
+                    {/* Domain restrictions */}
+                    <View style={[styles.row, styles.rowSpace]}>
+                        <LabelMedium tag="label" style={styles.domainMin}>
+                            {"domain min"}
+
+                            <Strut size={spacing.xxSmall_6} />
+                            <TextField
+                                type="number"
+                                style={styles.domainMinField}
+                                value={domainEntries[0]}
+                                onChange={handleDomainChange.bind(null, 0)}
+                            />
+                        </LabelMedium>
+                        <Strut size={spacing.medium_16} />
+                        <LabelMedium
+                            tag="label"
+                            aria-label="domain max"
+                            style={styles.domainMax}
+                        >
+                            {"max"}
+
+                            <Strut size={spacing.xxSmall_6} />
+                            <TextField
+                                type="number"
+                                style={styles.domainMaxField}
+                                value={domainEntries[1]}
+                                onChange={handleDomainChange.bind(null, 1)}
+                            />
+                        </LabelMedium>
+                    </View>
+
+                    {/* Actions */}
+                    <LockedFigureSettingsActions
+                        figureType={props.type}
+                        onMove={onMove}
+                        onRemove={onRemove}
+                    />
+                </PerseusEditorAccordion>
+            )}
+        </UniqueIDProvider>
+    );
+};
+
+const styles = StyleSheet.create({
+    accordionHeader: {
+        textOverflow: "ellipsis",
+        maxWidth: "calc(100% - 64px)",
```

Comment:
Why was it necessary to use `calc` here?  I think `padding` should work too, right?

Reply:
Explanation added.

Reply:
Thank you for the comprehensive explanations!